docs/use_cases/contribute_use_cases.rst

```diff
@@ -0,0 +1,112 @@
+============================================
+How to Contribute a Use Case & Sample Script
+============================================
+
+People use Parsons for a great many things. We like to document and share that usage to save work and inspire other members of the community.
+
+A "use case" is a narrative description of any length (but usually at least a couple paragraphs) explaining the problem the user faced and why and how they used Parsons. The sample script is a cleaned-up version of the script they used that someone else can then take and adapt.
+
+To submit a use case and sample script, please take the following steps. Please reach out to the community at any point if you have questions about these steps, need someone to pair with you on this process, or for any other reason.
+
+Steps:
+
+  1. Write up your use case in a file in ``docs/use_cases``. Use the guidelines below for style and content. The file should end with ``.rst``. (That means it's a `Restructured Text <https://www.writethedocs.org/guide/writing/reStructuredText/>`_ file. Note that the syntax for RST files can be a little tricky!)
```

Comment:
We might want to mention in this step that they can use the GitHub web editor, which will create a fork, since we mention later that they should make a PR.

Reply:
I put that in a separate note at the beginning. Not sure how visible the layout is when just looking at the PR, so attaching a screenshot:

![Screenshot from 2022-02-17 16-33-19](https://user-images.githubusercontent.com/1179362/154574496-2ffe39ef-c3c3-4660-a3ca-c667ec25d7d9.png)

Reply:
You didn't miss it, I added it after you made your suggestion! Ah the ambiguities of the English language. :)